stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/viewmodel/MessageInputViewModel.kt

```diff
@@ -46,7 +45,7 @@ public class MessageInputViewModel @JvmOverloads constructor(
      * Holds information about the current channel and is actively updated.
      */
     public val channelState: ChannelState =
-        chatClient.watchChannelAsState(cid, MessageListViewModel.DEFAULT_MESSAGES_LIMIT, viewModelScope)
+        chatClient.watchChannelAsState(cid, 0, viewModelScope)
```

Comment:
Can we use named parameters here or extract 0 to a constant? It might be hard to guess what does this magic number mean